# Camera -> IMU, LiDAR -> IMU (그대로 유지)
cam_to_imu:
  T: [[-0.00523798, -0.0398739, 0.99919099, 0.11545806],
      [-0.9999804, 0.00363629, -0.00509701, -0.00916812],
      [ -0.00343011, -0.9991981, -0.03989216, -0.03346498],
      [ 0.0,           0.0,           0.0,           1.0]]

lidar_to_imu:
  T: [[ 0.688536,  0.004840,  0.725186,  0.110294],
      [-0.010386,  0.999941,  0.003187, -0.002712],
      [-0.725128, -0.009726,  0.688545,  0.057693],
      [ 0.0,       0.0,       0.0,       1.0]]

# ---- 카메라 내부/왜곡 파라미터 ----
camera:
  # CameraInfo 토픽을 쓰지 않고, 아래 intrinsics/왜곡/해상도를 강제로 사용하려면 false
  use_camera_info: false

  # pinhole: intrinsics = [fx, fy, cx, cy]
  intrinsics: [634.2358506386614, 634.2909384110677, 631.4370690331413, 374.41418400883066]

  # radtan: [k1, k2, p1, p2]  (OpenCV는 [k1,k2,p1,p2,k3] 형태; k3는 0으로 패딩됨)
  distortion: [-0.049879140602722534, 0.04189503052568046, -0.0002581078253831348, -0.00036332730935522637]

  # 이미지 해상도
  resolution: [1280, 720]

# 시각화 옵션
max_range: 60.0
point_size: 2
draw_alpha: 0.9
